x-pack/plugins/ml/public/services/results_service.js

```diff
@@ -857,6 +867,10 @@ function getRecordsForInfluencer(jobIds, influencers, threshold, earliestMs, lat
         }
       });
     }
+    // TODO: might have to do some extra work around this query
```

Comment:
What extra work might be needed here?

Reply:
Just a comment from earlier that I missed removing - I've removed it 👍